HousingSearchApi/V1/Gateways/SearchPersonESHelper.cs

```diff
@@ -4,6 +4,7 @@
 using Amazon.Lambda.Core;
 using HousingSearchApi.V1.Domain;
 using HousingSearchApi.V1.Infrastructure;
+using HousingSearchApi.V1.Infrastructure.Sorting;
 using Nest;
 
 namespace HousingSearchApi.V1.Gateways
```

Comment:
Gateways folder seems a bit busy and it's difficult to understand what is the Gateway for my use case. Maybe subfolders might help to organize related things.

Reply:
Agreed - subfolder suggestion names?